_posts/2022-08-31-release-2.3.0.md

```diff
@@ -0,0 +1,92 @@
+---
+layout: post
+title: MDAnalysis 2.3 is out
+---
+
+We are happy to release the version 2.3.0 of MDAnalysis!
+
+This relatively small update to MDAnalysis reflects our commitment
+to doing more frequent (trimonthly) releases of the library.
+
+In line with [NEP 29][], the minmum required NumPy version has been
+raised to 1.20.0 (1.21.0 for macosx-arm64).
+
+Supported Python versions: **3.8, 3.9, 3.10**
+
+Supported Operating Systems:
+  - **Linux** (on [x86_64][], [aarch64][], and [ppc64le][] *note: aarch64 and ppc64le can only be installed via pip for now*)
+  - **Windows** (on [x864_64][])
+  - **MacOS** (on [x86_64][] and [Apple M1][])
+
+
+## Upgrading to MDAnalysis version 2.3.0
+
+To update with `conda` from the [conda-forge channel][] run
+
+```bash
+conda update -c conda-forge mdanalysis
+```
+
+To update from [PyPi][] with `pip` run
+
+```bash
+pip install --upgrade MDAnalysis
+```
+
+For more help with installation see the [installation instructions in the User Guide][].
+
+
+## Notable changes
+
+For a full list of changes, bugfixes and deprecations see the [CHANGELOG][].
+
+### Fixes:
+ - Fixed reading error when dealing with corrupt PDB CONECT records, and an issue where MDAnalysis would write out unusable CONECT records with index>100000 (Issue #988).
+
+### Enhancements:
+ - Formal charges are now read from PDB files and stored in a `formalcharge` attribute (PR #3755).
+ - A new normalizing `norm` parameter for the `InterRDF` and `InterRDF_s` analysis methods (Issue #3687).
+ - Improved Universe serialization performance (Issue #3721, PR #3710).
+
+### Changes:
+ - To install optional packages for different file formats supported by MDAnalysis, use `pip install ./package[extra_formats]` (Issue #3701, PR #3711).
+
+### Deprecations:
+ - The `extra_requires` target `AMBER` for `pip install ./package[AMBER]` will be removed in 2.4.0. Use `extra_formats` (Issue #3701, PR #3711).
+
+### CZI EOSS Performance Improvements:
+
+A series of performance improvements to the MDAnalysis library's backend have been made as per planned work under MDAnalysis' CZI EOSS4 grant. Further details about these will be provided in a future blog post.
```

Comment:
Didn't really want to go into details here given I assume more details will be given in the future.